containenv/containenv/cli/commands/init.py

```diff
@@ -48,3 +77,152 @@ def __call__(self):
         return run_tasks(
             'running some tasks',
             self.tasks)
+
+    def _make_task(self, method):
+        '''create a run_tasks compliant task'''
+        return (method.__doc__, method)
+
+    def make_path(self):
+        '''construct target project path'''
+        self.proj_path = os.path.abspath(self.metasource)
+        self.proj_name = os.path.basename(self.proj_path)
+
+    def _check_project_existence(self):
+        '''verify that self.proj_path is a valid project'''
+        if not os.path.isdir(self.proj_path):
+            error = ('init must be called on a project directory but the '
+                     'argument it received was: {}\nwhich does not map to '
+                     'a path to an existing directory.'
+                     ''.format(self.metasource))
+            raise ProjectDirectoryDoesNotExist(error)
+
+    def check_initialization_state(self):
+        '''determine whether a target project exists'''
+        self._check_project_existence()
+        if os.path.isdir(os.path.join(self.proj_path, '.containenv')):
```

Comment:
Yes, but I want to make it an explicit command, or subcommand this is tracked in #30.